lib/puppet/provider/service/systemd.rb

```diff
@@ -30,6 +30,20 @@ def enabled?
     begin
       systemctl("is-enabled", @resource[:name])
     rescue Puppet::ExecutionFailure
+      # The masked state is equivalent to the disabled state in terms of
```

Comment:
So, if I remember correctly, there was/is a bug where aliased commands would provide incorrect output to is-enabled.

However, I'm going to change this to use the LoadState and SubState properties together to make it a single system call.